lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -549,3 +563,29 @@ MITIGATION_02_CRI_DCMAW:
       targetState: PYI_ANOTHER_WAY
     enhanced-verification:
```

Comment:
Do you know which of these events corresponds to the 'ineligible/abandon' case in the UCD diagram?

In the main (non-mitigation) journey, we route all of these into the fallback passport/driving licence/f2f route - I wonder if we should be doing something similar here. I'm worried that it might need to be more than just the `enhanced-verification` case, but not 100% sure when each occurs.

Reply:
I had assumed this would present itself under the `not-found` event but maybe that's worth double checking with UCD folks

Reply:
Yeah, it feels like we need to understand
- What causes DCMAW to return the different codes (e.g. not-found vs. access-denied)
- Which of these UCD want to route to F2F mitigation vs. 'no match' vs. 'another way'

I'm afraid I'm not 100% sure of the answer to either! - Are you happy to try and find this out, or would you like me to reach out to them. We could also try Simon first 🤷 

cc @rpayal

Reply:
Agreed in discussion:
- pyi-no-match should route to 'no match' (handled in the CRI parent state)
- all the other error cases should route to 'suggest F2F' (unless F2F is disabled, in which case we go to 'another way')